tests/testPsfAttributes.py

```diff
@@ -36,16 +36,13 @@
 
 import lsst.afw.geom as afwGeom
 import lsst.meas.algorithms as measAlg
-import lsst.pex.logging as logging
 import lsst.utils.tests
 
 try:
     type(verbose)
     import lsst.afw.display.ds9 as ds9
 except NameError:
     verbose = 0
-    logging.Trace.setVerbosity("meas.algorithms.Interp", verbose)
-    logging.Trace.setVerbosity("afw.detection.Psf", verbose)
```

Comment:
Similar to ``testInterp.py``, there are debug messages from ``afw.image.Mask`` but no loggers of these names exist.